### ISSUE TITLE
Launching installed and portable instances at the same time is not possible

### ISSUE THREAD

**Issue by 1024mb** [stax76/mpv.net#263]
**Describe the bug**
I have two instances of mpv.net, one is installed and one is portable. If I launch one of them and then try to launch the other it doesn't launch it and instead the already open process is called, of course this is "more noticeable" when opening files. Normal installation is associated with all the video files so double clicking opens that instance but drag&dropping files onto the portable executable while the other instance is running opens those files there instead of running its own process. *That's a simplification, I don't open files that way, I have also two instances of XYplorer, one is configured to use the portable instance to open files the other is not.*

**To Reproduce**
1. Install and also use the portable versions
2. Open a file using one of them
3. Try to open a file using the other
4. File will open with the already running process

**Expected behavior**
Files should be opened in their own process if we are opening them with two different instances.

**Comment by hooke007** [comment]
https://github.com/stax76/mpv.net/blob/master/docs/Manual.md#--process-instancevalue

**Comment by 1024mb** [comment]
Yes, I have read and understand that (I think?) but shouldn't that apply for every instance/executable? The installed and portable versions both have separated executables, both have separated configurations, then why I can only open a single instance when they are supposed to be independent from each other? 
  
I know file associations are only possible with only one version at a time, so double clicking a file should open the version which is associated with and let us also open the version that is not associated with anything at the same time because they should be independent from each other.

**Comment by hooke007** [comment]
Because the default only allow one single process, so why not change that option to multi?

**Comment by 1024mb** [comment]
But wouldn't that mean that every time I double click a video file a new window will open?  
I want both versions to work with single, so if I double click a video it opens in the installed version, and if I drag&drop a file to the portable it opens another window then if I double click again on another video file I want that video file to be opened in the window of the installed version that is already running, same if I drag&drop another file to the portable's executable, I want that file to be opened in the (portable) process that is already running.

**Comment by hooke007** [comment]
Hmm....That sounds like the feature request rather than bug.

**Comment by 1024mb** [comment]
Yes, I wasn't sure what to pick, can you please change it?

**Comment by stax76** [comment]
It's implemented with a mutex and the ID 'mpvnetProcessInstance', so the ID is always the same.

In the next release, it will use the MD5 hash of the mpvnet.conf file path as ID.

**Comment by 1024mb** [comment]
@stax76 Thakn you!

**Comment by stax76** [comment]
You are welcome, there is a new beta version online.

**Comment by 1024mb** [comment]
> 
> 
> You are welcome, there is a new beta version online.

I have tried the beta and is kinda working. These are the problems I've found: 

- If I open two files each in a different instance they open in their own window but then if I open more files they all always open in the first started process, i.e. lets say the first file I opened was with the portable instance and the second one was with the installed one, if I open more files (no matter what instance I use, associations, drag&drop, command line -which is what I'm guessing XYplorer uses-) they all open in the portable window as it was the first process to start.
- Remember window size -which I've set to always- doesn't work anymore if the process exits. I've tested this with the two instances. If I open one file the window is set to the file's resolution and apparently to a max size which I'm guessing is 720p (I'm saying this because my monitor resolution is 1080p and I've opened a 1080p video file and the window didn't occupy the whole screen), if I open more files without closing the window the window size doesn't change but then if I close the window and open another file now the window changes again to the file's resolution, if I close and open another file, again, window size is the file's resolution. So it seems like a combination of video and with/height-session is being used for start-size.
